sale_timesheet_project_link/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Sale Timesheet Project Link
+===========================
+
+Simply adds a button on the sale order to be redirected on the linked projects.
+Linked projects are:
+
+#. Projects linked to order's analytic account
+#. Projects linked to order line's tasks
+#. Projects linked to the order line's product
```

Comment:
I don't see this last item in _compute_all_projects, is it obtained indirectly?

Reply:
The last item is given by the ```project_id``` field available in the ```sale_timesheet``` module.